README.md

```diff
@@ -423,6 +423,16 @@ def http_filter(request, response):
 iopipe = IOpipe(plugins=[TracePlugin(auto_http=True, http_filter=http_filter)])
 ```
 
+To add additional HTTP headers to your ttrace data use `http_headers`:
```

Comment:
Typo with `ttrace`.  (As someone with the occasional MBP stuck-key issue...solidarity.)